app/src/main/java/com/fastaccess/provider/timeline/TimelineProvider.java

```diff
@@ -99,12 +102,10 @@
                             .append(from)
                             .append(" ")
                             .bold(issueEventModel.getRequestedReviewer().getLogin());
-                } else if (event == IssueEventType.closed || event == IssueEventType.reopened) {
+                } else if (event == IssueEventType.closed) {
                     if (issueEventModel.getCommitId() != null) {
```

Comment:
I'll have to be on my laptop to see what this does do. 

_Sent from my Samsung SM-G950F using [FastHub Debug](https://play.google.com/store/apps/details?id=com.fastaccess.github)_